src/pages/OnboardingAccounting/BaseOnboardingAccounting.tsx

```diff
@@ -112,13 +126,44 @@ function BaseOnboardingAccounting({shouldUseNativeStyles}: BaseOnboardingAccount
                 text={translate('common.confirm')}
                 // eslint-disable-next-line rulesdir/prefer-early-return
                 onPress={() => {
-                    if (!userReportedIntegration) {
+                    if (userReportedIntegration === undefined) {
                         setError(translate('onboarding.purpose.errorSelection'));
                         // eslint-disable-next-line no-useless-return
                         return;
                     }
 
-                    // TODO: call CompleOnboarding API after the back-end PR is complete
+                    if (!onboardingPurposeSelected) {
+                        return;
+                    }
+
+                    if (onboardingPolicyID) {
+                        Policy.enablePolicyConnections(onboardingPolicyID, true);
+                    }
```

Comment:
Same ^ This should always be available

Reply:
We should not call this. It will be done in the backend already.

Reply:
@marcaaron Should we update the optimistic data for policy that will enable connection if `userReportedIntegration` is `undefined`?

Reply:
yes, you've got the idea. If it has a value then we can enable it optimistically. But we don't need  the navigation logic from `enablePolicyConnections()` just the `onyxData` parts.

Reply:
>enable connection if userReportedIntegration is undefined?

But I think you meant _not_ `undefined`?

Reply:
> But I think you meant not undefined?

Yes, I meant that 😄

Reply:
@nkdengineer Shouldn't we remove that line already? (the call to `Policy.enablePolicyConnections`)

Reply:
Ah, I missed it.